hazelcast-all/pom.xml

```diff
@@ -164,7 +164,7 @@
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast-hibernate5</artifactId>
-            <version>1.3.2</version>
+            <version>1.4.0-SNAPSHOT</version>
```

Comment:
We must remember to release this before GA :)

Reply:
Yes, it's on my todo list to remind Jiri about that additional release step once he is back from vacation.